setup.py

```diff
@@ -49,6 +49,7 @@ def find_version(*file_paths):
         "requests>=2.22",
         "hypothesis>=4.32",
         "colorama>=0.4.1",
+        "docker>=4.3.1",
```

Comment:
This line conflicts with https://github.com/aws/aws-sam-cli/blob/develop/requirements/base.txt#L11 resulting in conflicts when using this with sam-cli.

Reply:
reported here as well: https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/issues/147

Reply:
tbh [SAM CLI requirements are unreasonably tight](https://github.com/aws/aws-sam-cli/blob/a8525fc8157d507c4b102477ded4d221deaed145/requirements/base.txt#L11) because [they insist it shouldn't be installed with pip](https://github.com/aws/aws-sam-cli/issues/1424)